web/static/js/components/room.jsx

```diff
@@ -38,9 +61,9 @@ class Room extends Component {
     return (
       <section className={styles.wrapper}>
         <div className={`ui equal width padded grid ${styles["category-columns-wrapper"]}`}>
```

Comment:
unrelated to your change, but:
if you pass css loader an [option](https://github.com/webpack-contrib/css-loader#camelcase) to camel case class names, you could write styles.categoryColumnsWrapper.